spiketoolkit/sorters/basesorter.py

```diff
@@ -0,0 +1,116 @@
+"""
+Here a proposal for the futur Sorter with class approach.
+
+The main idea is to decompose all intermediate steps to get more 
+flexibility:
+  * setup the recording (traces, output folder, and so...)
+  * set parameters
+  * run the sorter (with futur possibility to make it in separate env/container)
+  * get the result (SortingExtractor)
+
+One benfit shoudl to compare the "run" time between sorter without
+the setup and getting result.
+
+One new idea usefull for tridesclous and maybe other sorter would
+a way to adapt params with datasets.
+
+
+"""
+
+import time
+import copy
+from pathlib import Path
+
+class BaseSorter:
+    
+    sortername = '' # convinience for reporting
```

Comment:
sortername --> sorter_name

Reply:
Just want to be consistent with variable names (this is small)

Reply:
OK